src/chat.rs

```diff
@@ -2804,7 +2806,47 @@ pub(crate) async fn create_send_msg_jobs(context: &Context, msg: &mut Message) -
         return Ok(Vec::new());
     }
 
-    let rendered_msg = match mimefactory.render(context).await {
+    // render message and pre message.
+    // pre message is a small message with metadata
+    // which announces a larger message. Large messages are not downloaded in the background.
```

Comment:
```suggestion
    // which announces a larger full message. Full messages are not downloaded in the background.
```
It's better to use the same terms everywhere. "Large message" could mean e.g. a big message from oher MUA.